docs/application/dotnet/TOC.md

```diff
@@ -1,5 +1,8 @@
 # Getting Started
 ## [Creating Your First Tizen .NET Application](/application/dotnet/getting-started/first-app.md)
+# [Managed Runtime Advantages](/application/dotnet/getting-started/runtime.md)
```

Comment:
There is a rule to make a menu tree of LNB (Left Navigation Bar).

# 1st Heading (<= You don't provide links because it has children)
## 2nd Heading (= 1st heading's child)
### [3rd Heading (= 2nd heading's child) ](1231.md)
### [3rd Heading (= 2nd heading's child) ](1232.md)
## 2nd Heading (= 1st heading's child)(12.md)
## 2nd Heading (= 1st heading's child)(13.md)

@hj-na-park
Who you are? I don't know if you don't fill your name in profile.
Furthermore, I don't know why you change the  docs/application/dotnet/index.md.
The content of a few files is too short and some files  are not for getting-started. It mean the files have to be moved to the other directory not docs/application/dotnet/getting-started/, as I told you before.

Reply:
Q) I don't know if you don't fill your name in profile. -> My name is Hyunjin Park.
Q)  I don't know why you changed the docs/application/dotnet/index.md.
Do you think index.md is too long, so you worry that developers may not read it?
- I got feedback from API team to improve tizen .net app development environment and made some changes.
My team members expressed opinions that developers were not comfortable reading because the page was too long. It is also different from the documentation of other development languages, so I decided to fit the configuration.
Q) The contents of a few changed files are too short and some files are not for getting-started. It mean the files have to be moved to the other directory not docs/application/dotnet/getting-started/, as I told you before.
- Kwon Ho-bum will add documents later. The shorter the content will be improved in the future. I would like you to give me an opinion on where you would like to move a document that is not relevant to `get started` in the current document.

Reply:
Q) Do you think index.md is too long, so you worry that developers may not read it?
A) I don't think so.
The current docs.tizen.org is not  final layout. Every page will provide TOC on the right top. TOC here is different of TOC.md. The build system will generate TOC automatically. If the feature is supported, a user will be able to access each section easily.

Its structure is similar with Overview of  Tizen Native Application(https://docs.tizen.org/application/native/index). 

Your plan) Kwon Ho-bum will add documents later. The shorter the content will be improved in the future. I create new directory and move files to it. Please check it.
A) It's not too late to change overall structure when Mr. Kwon Ho-bum improve contents.
How about change it later? If so, we can investigate all changed contents and restructure menu tree.

Reply:
Kwon Ho-bum agreed with your opinion. We will make the changes later.